.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -66,22 +71,35 @@ stages:
       - script: |
           echo "*** Setting loadTestTargetFQDN to $(frontdoor_fqdn)"
           echo "##vso[task.setvariable variable=azureLoadTestTargetFQDN]$(frontdoor_fqdn)"
-        displayName: 'Set Loadtest targetUrl to Azure Front Door'
 
-    # Set loadTestTargetFQDN to the FQDN specified in the pipeline parameters
+          echo "*** Setting loadTestBaseline to $(azureLoadTestBaseline)"
+          echo "##vso[task.setvariable variable=loadTestBaseline]$(azureLoadTestBaseline)"
+
+        displayName: 'Set Azure Load Test variables'
+
+    # override baseline when chaos testing is enabled
+    - ${{ if eq(parameters.runChaosTesting, 'true') }}:
+      - script: |
+          echo "*** Setting loadTestBaseline to $(chaosLoadTestBaseline)"
+          echo "##vso[task.setvariable variable=loadTestBaseline]$(chaosLoadTestBaseline)"
+        displayName: 'Override loadTestBaseline for Chaos Experiments'
+
+    # setting parameters for embedded load tests
```

Comment:
??
if ne(parameters.embeddedLoadTest, 'true')  ==> # setting parameters for embedded load tests

Reply:
should it be "parameters for standalone load tests"?